.github/ISSUE_TEMPLATE/dotnet-breaking-change.md

```diff
@@ -32,6 +32,13 @@ assignees: ''
 
 ### New behavior
 
+### Category
+
+<!-- Choose one of the following compatibility categories: -->
+
+- [ ] Binary compatibility (the newer version can be used without recompiling)
+- [ ] Source compatibility (successfully recompiling against newer version requires source changes)
```

Comment:
"requires source changes" sounds like **in**compatibility (?)

Reply:
The idea is that the breaking change affects source compatibility. Does that make sense?

Reply:
Yes, it's OK as-is, it just seems out of sync with the explanatory phrase for binary compatibility, i.e., one identifies compatibility and one identifies incompatibility.

Reply:
I see. I've made them consistent now, thanks.